go/inst/oracle_gtid_set_entry.go

```diff
@@ -24,51 +24,233 @@ import (
 )
 
 var (
+	// Regex Pattern to match a Singe Value Interval e.g. 1 in uuid:1
 	singleValueInterval = regexp.MustCompile("^([0-9]+)$")
-	multiValueInterval  = regexp.MustCompile("^([0-9]+)[-]([0-9]+)$")
+
+	// Regex Pattern to match a Multi Value Interval e.g. 1-5 in uuid:1-5
+	multiValueInterval = regexp.MustCompile("^([0-9]+)[-]([0-9]+)$")
+
+	// Regex Pattern to match GTID tags
+	// Tag must start with a letter e.g. tag1 in uuid:tag1:1-5
+	tagRegex = regexp.MustCompile("^[a-z_][a-z0-9_]{0,31}")
 )
 
+type TagInterval struct {
```

Comment:
Just an observation:
I know that Orchestrator codding convention doesn't care about types visibility, making almost all types in all packages public, but do we need this type to be visible outside `inst` package?

BTW, below we have `OracleGtidSetEntry` which probably could be a private type as well...

Reply:
Done. Changed the `TagInterval` struct to private.